stablehlo/dialect/TypeInference.cpp

```diff
@@ -986,6 +987,13 @@ LogicalResult inferOptimizationBarrierOp(
   return success();
 }
 
+LogicalResult inferOutfeedOp(Optional<Location> location, ValueRange inputs,
+                             Value token, StringAttr outfeed_config,
+                             SmallVectorImpl<Type>& inferredReturnTypes) {
+  inferredReturnTypes.emplace_back(token.getType());
```

Comment:
Since this function does not use `inputs` and `outfeed_config` should these be removed or included in case future changes require them?

Reply:
Yes, let's remove for now.